apps/app/src/components/Vault/VaultPageButtons.tsx

```diff
@@ -1,15 +1,20 @@
 import { Vault } from '@generationsoftware/hyperstructure-client-js'
-import { DepositButton, WithdrawButton } from '@shared/react-components'
+import { GiftIcon } from '@heroicons/react/24/solid'
+import { DelegateButton, DepositButton, WithdrawButton } from '@shared/react-components'
+import { Intl } from '@shared/types'
 import classNames from 'classnames'
 import { useTranslations } from 'next-intl'
 
 interface VaultPageButtonsProps {
   vault: Vault
   className?: string
+  intl?: {
+    common?: Intl<'delegate'>
+  }
 }
 
 export const VaultPageButtons = (props: VaultPageButtonsProps) => {
-  const { vault, className } = props
+  const { vault, className, intl } = props
```

Comment:
Passing localization as props is generally pretty annoying, so we should only do it when necessary (when in a shared library). Here we can just use the translations directly.